Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py

```diff
@@ -41,6 +41,32 @@ def testAtomPairTypes(self):
     self.assertTrue(rdMD.GetAtomPairAtomCode(mol.GetAtomWithIdx(1),2)==\
                     0 | (2 | 1<<params.numPiBits)<<params.numBranchBits)
 
+  def testAtomPairTypesChirality(self):
+    mols = [Chem.MolFromSmiles(x) for x in ("CC(F)Cl","C[C@@H](F)Cl","C[C@H](F)Cl")]
+    self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)),
+        rdMD.GetAtomPairAtomCode(mols[1].GetAtomWithIdx(1)))
+    self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)),
+        rdMD.GetAtomPairAtomCode(mols[2].GetAtomWithIdx(1)))
+    self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)))
+    self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[1].GetAtomWithIdx(1),includeChirality=True))
+    self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[2].GetAtomWithIdx(1),includeChirality=True))
+    self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[1].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[2].GetAtomWithIdx(1),includeChirality=True))
```

Comment:
To make the intention of the test clearer, I would use a loop:
```
for i1, i2 in (0, 1), (0,2):
  self.assertEqual(rdMD.GetAtomPairAtomCode(mols[i1].GetAtomWithIdx(1)),
                   rdMD.GetAtomPairAtomCode(mols[i2].GetAtomWithIdx(1)))

# Including chirality, atom codes are no longer the same
self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1), includeChirality=True),
                 rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)))
for i1, i2 in (0, 1), (0,2), (1, 2):
  self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[i1].GetAtomWithIdx(1), includeChirality=True),
                      rdMD.GetAtomPairAtomCode(mols[i2].GetAtomWithIdx(1), includeChirality=True))
```
and maybe even clearer pre-calculating the atoms
```
atomMol = [mols[i].GetAtomWithIdx(1) for i in range(3)]
for i1, i2 in (0, 1), (0,2):
  self.assertEqual(rdMD.GetAtomPairAtomCode(atomMol[i1]),
                   rdMD.GetAtomPairAtomCode(atomMol[i2]))

# Including chirality, atom codes are no longer the same
self.assertEqual(rdMD.GetAtomPairAtomCode(atomMol[0], includeChirality=True),
                 rdMD.GetAtomPairAtomCode(atomMol[0]))
for i1, i2 in (0, 1), (0,2), (1, 2):
  self.assertNotEqual(rdMD.GetAtomPairAtomCode(atomMol[i1], includeChirality=True),
                      rdMD.GetAtomPairAtomCode(atomMol[i2], includeChirality=True))
```